Esterni/ManualeUtente/components/c4-Istruzioni_per_utilizzo-MU.tex

```diff
@@ -1,50 +1,333 @@
 \section{Istruzioni per l'utilizzo}
```

Comment:
Non so se solo io, ma per far contento tullio metterei le cose in questo ordine:

- 4.1
- 4.2
- 4.6
- 4.7
- 4.3
- 4.4
- 4.5
- 4.8
Insomma per non avere tullio che dice che abbiamo una visione a finestre, come ho detto io la logica sarebbe "Ecco come si avvia, ecco la schermata principale, ecco cosa puoi fare ed ecco che azioni puoi effettuare" e solo in seguito il resto della gui

Reply:
Ok fatto